### PR TITLE
Feature/for review

### PR DESCRIPTION
**Pull request by RichardSyfer**

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require './library'
+
+library = Library.new
+library = File.exist?('./db/db.yml') ? library.load_dump : library.first_load
+# library.first_load
```
Comment (Svatok) [inline main.rb]: remove comment

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require './library'
+
+library = Library.new
+library = File.exist?('./db/db.yml') ? library.load_dump : library.first_load
```
Comment (Svatok) [inline main.rb]: `library.first_load` not works
check `Date.today`

**Review by Svatok — COMMENTED** [review]
1. Put files with entities into folder lib (or something like this).
2. Check code with rubocop

Good work! :+1:

File: main.rb (hunk 2 above)
Comment (RichardSyfer) [reply main.rb]: 1. Can't reproduce problem with "#first_load"
2. Added require 'date'

**Review by Svatok — APPROVED** [review]

**Pull request merged**